xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+   2. I've asked Tanja, and we agreed, the term should NOT be repeated
+      in the definition. For example, in the "Registry" definition, you should
+      remove the "A registry is..." and write "Storage for already created
+      images." Same for the other terms.
+  </remark>
+
+  <variablelist>
+   <varlistentry xml:id="Docker_engine">
+    <term>&deng;</term>
+    <listitem>
+     <para>
+      &deng; is a server-client type application that performs all
+      tasks related to virtual machines. &deng; comprises the
+      following:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <formalpara>
+        <title>Daemon:</title>
+        <para>
+         The server side of &deng; manages all &docker; objects (images,
+         containers, network connections used by containers, etc.).
+        </para>
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>REST API:</title>
+        <para>
+         Applications can use this API to communicate directly with the daemon.
+        </para>
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>CLI Client:</title>
+        <para>
+         Enables you to communicate with the daemon. If the daemon is running
+         on a different machine than the CLI client, the CLI client can
+         communicate by using network sockets or the REST API provided by
+         &deng;.
+        </para>
+       </formalpara>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="Docker_Image">
+    <term>Image</term>
+    <listitem>
+     <para>
+      An <emphasis>image</emphasis> is a read-only template<remark>toms 2019-02-04: 
+      template sounds a bit strange in this context.
+      On Quora.com, I've found an interesting definition:
+      "An image is a compressed, self-contained piece of software which has
+      to be unwrapped in order to use the functionality." Maybe you can add
+      it somehow.</remark> used to create a
+      <emphasis>virtual machine</emphasis> on the host server. A &docker; image is
+      made by a series of layers built one over the other. Each layer
+      corresponds to a permanent change, for example an update of an
+      application.
+      <remark>toms 2019-02-04: The following sentence is dangerous and
+      misleading IMHO. The term "changes" does not relate to a Dockerfile.
+      That file is used merely as a kind of "build recipe".
+      I would propose to rephase/move the sentence at the beginning of this
+      paragraph:
+      "An image is a read-only template used to create a virtual machine on
+      the host server. Such an image is represented/build by using a
+      Dockerfile which acts as an "recipe". A Docker image is..."
+      </remark>
+      The changes are stored in a file called a
+      <emphasis>Dockerfile</emphasis>. For more details see
```

Comment:
Because the sentence before is about the Dockerfile, I would mention that following link is about the Docker image:
"For more details on Docker images, see  [...]."

Reply:
My previous comment applies here....containers are not virtual machines, images are not templates, they are simply files. A dockerfile is more like a manifest vs a recipe...it describes the building of the image, but doesn't have the parts to actually create the file.

Reply:
@taroth21 changed.